docs/registration/registration-reject-pa.md

```diff
@@ -1,31 +1,28 @@
 ---
-title: Registration - Reject of PA
+title: People Affected - Reject of PA
 hide:
   - toc
 ---
 
-!!! info "This section explains"
-    Steps to reject PAs
-
 Not everyone registered for your program may be eligible to be included in the program, or there may be cases of duplicate registrations. In these cases you may need to decline People Affected from the program.
 
 You use this page and dropdown menu to choose the appropriate next actions : validating, rejecting or editing registration of PA, move the PA into inclusion phase.
 
 
-### Decline People Affected from the program
+### **Decline People Affected from the program**
 
 The People Affected table shows you all of the registered households for your program. Their status should be **Registered**.
 
 - Decline People Affected from the cash program by clicking on **Choose Action : Mark as Declined**
 - Then clicking, a tick box :material-checkbox-blank-outline: appears on the left of the PA name column.
-  - Make an individual selection by clicking on the :material-checkbox-outline: on the PA line;
-  - Or a bulk selection by clicking on the :material-checkbox-multiple-marked-outline: on the upper line **Select**
+  - Make an **individual selection** by clicking on the :material-checkbox-outline: on the PA line;
```

Comment:
see notes above, in some cases these items are bold and in some cases are not. Update to keep consistent.